recipe-client-addon/lib/ClientEnvironment.jsm

```diff
@@ -174,6 +179,21 @@ this.ClientEnvironment = {
       return Preferences.get("privacy.donottrackheader.enabled", false);
     });
 
+    XPCOMUtils.defineLazyGetter(environment, "experiments", async () => {
```

Comment:
These are being set on the return value of `getEnvironment`, which is called once per filter expression we evaluate (we actually should probably optimize that to run once per call to `run` instead). So they aren't shared between filter expression evals, nor between calls to `run` every 24 hours.